inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/metric/MetricsCollector.java

```diff
@@ -15,10 +15,9 @@
  * limitations under the License.
  */
 
-package org.apache.inlong.sort.pulsar.withoutadmin;
```

Comment:
you remove it in pulsar pom file when you move this class file
```
  <dependency>
            <groupId>org.apache.inlong</groupId>
            <artifactId>sort-format-base</artifactId>
            <version>${project.version}</version>
            <scope>provided</scope>
        </dependency>
```

Reply:
The compilation will fail since it depences on sort-format-base

Reply:
the scope of sort-format-base in connector-base is provided

Reply:
The format-base in connector-base is provided scoped, I changed to compile then

Reply:
> The format-base in connector-base is provided scoped, I changed to compile then

@EMsnap kafka pom file also add it. You also need remove it.
```
  <dependency>
            <groupId>org.apache.inlong</groupId>
            <artifactId>sort-format-base</artifactId>
            <version>${project.version}</version>
            <scope>provided</scope>
        </dependency>
```